src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -0,0 +1,337 @@
+
+<Callout variant="important">
+  Session replay is in limited preview. While not everyone can access it just yet, you can request access to the preview [here](https://newrelic.com/platform/session-replay-early-access).
+</Callout>
+
+Session replay allows you to play back user interactions on your web app and map other telemetry data to help you understand your user's journeys. Session replay can be used for troubleshooting and improving the end user experience, such as:
+
+* **Troubleshooting JavaScript errors**: Troubleshoot the cause of an error by showing you what the user was doing when the error occurred. For example, if an end user receives an error while trying to complete an ecommerce transaction, you can use session replay to see which steps they took and what data they entered before the error occurred. This can help you quickly identify and fix the root cause of the problem.
+* **Improving the user experience**: Discover areas of your web app that are causing users frustration. For example, you might see that users are getting lost in your navigation or clicking on buttons that don't do anything.
+* **Identifying issues in key user journeys**: Identify areas of your web app that are causing performance problems. For example, if session replay reveals the `/cart/checkout` page is experiencing excessive page load time, you'd prioritize fixing this critical user journey.
+
+Here's an example of how a session replay looks:
+
+<img
+    title="Session replay details"
+    alt="Screenshot of session replay details"
+    src={browserSessionReplayDetails}
+/>
+
+    <figcaption>
+  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Session replay**: Visualize your end user's experience and correlate issues in their journey with sessiont traces.
+</figcaption>
+
+## How session replay works [#how-it-works]
+
+Session replay works by taking a snapshot of the end-user's DOM (Document Object Model) when the web page loads. The DOM is a hierarchical representation of all the elements on a web page, such as the text, images, and buttons. The browser agent uses a library to record DOM changes as the user interacts with the page. This data is then stored in New Relic's database and can be replayed up to 8 days after the session occurred. The replay is pixel perfect, and you can see exactly what the user's screen looks like when an error occurs.
+
+### FAQ [#faq]
+
+<CollapserGroup>
+<Collapser
+    id="user-data-collection"
+    title="How is user data collected while maintaining privacy?"
+>
+    * Minimal data storage: Session replay does not store screenshots or videos. It only records the essential DOM state changes required to recreate the user's experience. This results in a much smaller amount of data being stored.
+    * Client-side privacy enforcement: Privacy settings are applied directly on the client-side, ensuring strong protection of user data from the outset.
```

Comment:
"client-side privacy" seems a bit market-y to me. If this is talking about how NR users can set their own privacy settings, I'd phrase it a bit different. "Client" seems a bit tense.